umap/templates/umap/map_init.html

```diff
@@ -1,7 +1,9 @@
 {% load umap_tags %}
 <div id="map"></div>
 <!-- djlint:off -->
-<script type="text/javascript">
-  let MAP = new L.U.Map("map", {{ map_settings|notag|safe }})
+<script defer type="text/javascript">
+  window.addEventListener('DOMContentLoaded', (event) => {
+    let MAP = new L.U.Map("map", {{ map_settings|notag|safe }});
+  });
```

Comment:
Does it make sense to have it twice in map_fragment and map_init?

Reply:
Solved with #1516